src/emulator/controller.ts

```diff
@@ -968,11 +970,28 @@ export async function startAll(
    * app hosting emulator may depend on other emulators (i.e auth, firestore,
    * storage, etc).
    */
-  const apphostingConfig = options.config.src.emulators?.[Emulators.APPHOSTING];
+  const apphostingEmulatorConfig = options.config.src.emulators?.[Emulators.APPHOSTING];
 
   if (listenForEmulator.apphosting) {
+    const rootDirectory = apphostingEmulatorConfig?.rootDirectory;
+    const backendRoot = resolveProjectPath({}, rootDirectory ?? "./");
+
+    // It doesn't seem as though App Hosting emulator supports multiple backends, infer the correct one
+    // from the root directory.
```

Comment:
File an issue saying we should add backendId to the emulators.apphosting config and add TODO